auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -78,6 +81,7 @@
             @add-failed-passcode-claimed="showPasscodeClaimedModal()"
             @add-unknown-error="showUnknownErrorModal('business')"
             @cancel="cancelAddBusiness()"
+            @businessIdentifier="businessIdentifier = $event"
```

Comment:
it took me for a moment to realize businessIdentifier is an event.. may be a 'event' name is better like businessIdentifier-changed or businessIdentifier-set or something will be better?

Reply:
Good point. Fixed.